### ISSUE TITLE
Unable to access notes. It showing "Unable to render code block".

### ISSUE THREAD

**Issue by PrajwalPatel** [kunal-kushwaha/DSA-Bootcamp-Java#1158]

**Comment by Coderjai01** [comment]
Not able to get access of the notes and pdfs becoz of above mentioned error

**Comment by AMAY369** [comment]
In which lecture, you guys are facing this issue ?

**Comment by akshatsha55** [comment]
> In which lecture, you guys are facing this issue ?

From the starting of the course...the notes aren't accessible.

**Comment by AMAY369** [comment]
![image](https://github.com/kunal-kushwaha/DSA-Bootcamp-Java/assets/105373537/3bdbbc29-17c4-4427-872d-14b4980729cb)

Try refreshing the window and then reloading the pdf

**Comment by AMAY369** [comment]
if you still are not able to access the pdf then download the pdf first and you will have it in your download folder
![image](https://github.com/kunal-kushwaha/DSA-Bootcamp-Java/assets/105373537/18231287-453e-42a4-ab59-c3cb83b1508a)

**Comment by PrajwalPatel** [comment]
![ss](https://github.com/kunal-kushwaha/DSA-Bootcamp-Java/assets/83801651/a19662c8-5266-4be0-8c0d-a24ebb4d5d83)
still not able to access...

**Comment by kondapalli19** [comment]
![image](https://github.com/kunal-kushwaha/DSA-Bootcamp-Java/assets/101511737/5ae5c5eb-338c-4468-8558-8ec8125371f1)
Cannot access it

**Comment by Coderjai01** [comment]
> if you still are not able to access the pdf then download the pdf first and you will have it in your download folder ![image](https://user-images.githubusercontent.com/105373537/269985359-18231287-453e-42a4-ab59-c3cb83b1508a.png)

No....this feature will not work until the pdf is not shown...now i am able to see this....after making the clone of this repository

**Comment by kondapalli19** [comment]
I'm still not able to access the nodes even after cloning

**Comment by kondapalli19** [comment]
Thanks for the help @Coderjai01  after cloning the repo I can access the pdf

**Comment by Coderjai01** [comment]
> Thanks for the help @Coderjai01 after cloning the repo I can access the pdf

Welcome .... every one can do this...make a clone of this repo you can easily access to the notes and pdf
